manta-js/package/src/BaseWallet.ts

```diff
@@ -174,4 +171,31 @@ export default class BaseWallet implements IBaseWallet {
       config.apiTimeout,
     );
   }
+
+  async wrapWalletIsBusy<T>(
```

Comment:
Move the walletIsBusy logic to BaseWallet, because isBusy is for the entire Context, not for one of the Pallets